public/src/js/models/article/editor.js

```diff
@@ -146,22 +168,66 @@ export default class Editor extends BaseClass {
         this.meta(value === this.field() ? undefined : value.replace(rxScriptStriper, ''));
     }
 
+    writeImage(value) {
+        var checkedValue = this.field() === value ? undefined : value.replace(rxScriptStriper, '');
+        this.meta({
```

Comment:
Because this is the only place where we update the `meta` we can simplify the logic. Instead of doing `this.meta` and subscribing to the value change, we can call `validateImage` directly.

We can get rid of a subscription and we never store a possibly invalid value in the meta.